src/components/pages/Cases/PDFOverviewExport/ExportGenerator.jsx

```diff
@@ -59,17 +59,22 @@ const styles = StyleSheet.create({
 });
 
 export function MyDoc(props) {
+  // case_date has a UTC format from the database
+  // convertDate/date formats it to a correctly configured date string
+  const convertDate = new Date(props.caseData[0].case_date);
+  const date = convertDate.toLocaleDateString();
+
   return (
     <Document>
       <Page style={styles.page}>
         <Image source={hrfLogo} style={styles.logo} />
         <View style={styles.table}>
           <View style={styles.tableRow}>
             <View style={styles.tableCol}>
-              <Text style={styles.columnNameCell}>Case ID</Text>
+              <Text style={styles.columnNameCell}>Case #</Text>
             </View>
             <View style={styles.tableCol}>
```

Comment:
When exporting the PDF to the user, perhaps we can change the "Nation of Origin" field to "Country of Origin" so the language is consistent with the table on Saved Cases

i.e. `    <Text style={styles.columnNameCell}>Country of Origin</Text>`

Reply:
I agree.